.ci-builds/.azure-pipelines.yml

```diff
@@ -703,7 +703,19 @@ jobs:
         mkdir -p icuexportdata/collation/implicithan
         ./bin/genrb -X -s data/coll/ --ucadata data/in/coll/ucadata-implicithan-icu4x.icu -d icuexportdata/collation/implicithan $FILES
         rm icuexportdata/collation/implicithan/*.res
-      displayName: 'Build collation data export files'
+      displayName: 'Build collation data files'
+      env:
+        LD_LIBRARY_PATH: lib
+    - script: |
+        cd icu4c/source
+        mkdir -p icuexportdata/segmenter/dictionary
+        for FILE in `ls data/brkitr/dictionaries`
+        do
+          ./bin/gendict --uchars data/brkitr/dictionaries/$FILE /dev/stdout | \
+          python3 -c 'import sys; data = sys.stdin.buffer.read(); print(f"trie_data = {[data[i + 1] << 8 | data[i] for i in range(64, len(data) - 1, 2)]}")' \
```

Comment:
Hmm, I would just say to copy in the .dict files without converting them to text, but I'm not sure if they are platform-independent. @markusicu should know.

Reply:
But I get that our files are mostly text.

Note that the thing that's more in the spirit of icuexportdata is to export the contents of the dictionary, optionally paired with that same data compiled into a data structure.

Reply:
It's not platform-independent, it's a list of `u16`s.

Most files in `icuexportdata` are pretty processed, like prebuilt tries and the like.